index.bs

```diff
@@ -288,6 +288,7 @@ The Web Authentication API is defined by the union of the Web IDL fragments pres
 
 <pre class="idl">
     partial interface Navigator {
+        [SecureContext]
```

Comment:
Should this be on the definition of interface WebAuthentication also (or instead)? Also @mikewest seemed to suggest adding this to other interfaces as well, such as ScopedCredentialInfo and so on. I'm not sure what that would do, but it's a thought.